community/cypher/runtime-util/src/main/scala/org/neo4j/cypher/internal/runtime/planDescription/LogicalPlan2PlanDescription.scala

```diff
@@ -405,6 +405,13 @@ case class LogicalPlan2PlanDescription(readOnly: Boolean, cardinalities: Cardina
             val lessThanBoundsText = lessThanBounds.bounds.map(bound =>
               s"<${bound.inequalitySignSuffix} ${bound.endPoint.asCanonicalStringVal}").toIndexedSeq
             (name, InequalityIndex(label.name, propertyKey, greaterThanBoundsText ++ lessThanBoundsText))
+          case PointDistanceSeekRangeWrapper(PointDistanceRange(point, distance, inclusive)) =>
+            val poi = point match {
+              case FunctionInvocation(Namespace(List()), FunctionName("point"), _, Seq(MapExpression(args))) =>
+                s"point(${args.map(_._1.name).mkString(",")})"
```

Comment:
The intention here was a short and readible 'other' column. In this case we will have something that looks like `point(x,y,crs)`, with the values stripped out because they are visually cluttering (include stuff like `  AUTOVAR32`) and are duplicates of other rows. The syntax `_._1.name` takes the tuple from the map, uses the key (`_1`) and only the `name` of the key.